Update README.md

dferhadian opened the pull request:
Here's my assignment for this week
Damien Ferhadian

dferhadian committed:
Update README.md

leej3 commented:
Sounds good. You 2 distinct steps, the latter more open ended to allow you to do more if you  have the time. Should work out nicely. Remember to follow the rubric as you put it together.

dferhadian committed:
Add files via upload

dferhadian committed:
Create project.py

dferhadian committed:
Rename project_spring_2020/project.py to get_volumes/get_volumes.py

dferhadian committed:
Rename project_spring_2020/Data.csv to get_volume/Data.csv

dferhadian committed:
Rename get_volume/Data.csv to get_volumes/Data.csv

dferhadian committed:
Rename project_spring_2020/sample_file.py to get_volumes/sample_file.py

dferhadian committed:
Create setup.py

dferhadian committed:
Update setup.py

dferhadian committed:
Rename get_volumes/Data.csv to get_volumes/Data/Data.csv

dferhadian committed:
Create get_volumes_test.py

dferhadian committed:
Update get_volumes.py

dferhadian commented:
Hi John,
You showed me some example for my test last thursday on Zoom and I can't find it anymore  (I thought  it would stay in the damien_demo repository you  created but it isn;t there anymore, could you help me with this?

leej3 commented:
I put it here: https://github.com/biof309/project_spring_2020/tree/damien_demo

dferhadian commented:
That's where I went but I don't see anything 
There are just the files without anything  written in it (it says for each folder and file that the  last contribution was done 2 months ago)

leej3 commented:
Weird, don't know why that wasn't pushed. Fixed now.

dferhadian committed:
Update sample_test.py

dferhadian committed:
Delete get_volumes_test.py

dferhadian committed:
Rename get_volumes.py to core_function.py

dferhadian committed:
Rename get_volumes/Data/Data.csv to tests/Data/Data.csv

dferhadian committed:
Delete sample_file.py

dferhadian committed:
Rename sample_test.py to get_volume_test.py

dferhadian committed:
Update get_volume_test.py

dferhadian committed:
Delete Data.csv

dferhadian committed:
Add files via upload

dferhadian committed:
Rename tests/Data.csv to tests/Data/Data.csv

dferhadian committed:
Update core_function.py

dferhadian commented:
Thank you so much John!

dferhadian committed:
Update setup.py

dferhadian committed:
Update README.md

dferhadian committed:
Update get_volume_test.py

dferhadian committed:
Updated test

dferhadian committed:
Update setup.py

dferhadian committed:
Update setup.py

dferhadian committed:
Update setup.py

leej3 committed:
fix data path

__file__ is a variable for the file that python is currently executing
useful in this case because the current working can be different
depending on where you ran your tests from

leej3 commented:
nice job. some issues with paths during testing that it seems many others had. With this you should be passing tests. congrats

dferhadian committed:
Merge pull request #1 from leej3/path_fix

quick fix

dferhadian commented:
Thank you john!

emilyyaklich commented:
Hi Damien,

Well done on the project. Pandas is a useful tool for working with datasets and you employ it well in your code. For your test, the [*pandas.testing.assert_frame_equal*](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html) might be helpful to you. Overall, great work! 

Cheers,

Emily
